spec/features/admin/admin_can_change_its_user_properties_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+describe "admin", type: :feature do
+
+  before :each do
+    @user = User.create(full_name:"Andrew Fink", email:"andrewfink@gmail.com", password:"password", role: 1)
+    visit login_path
+    fill_in('email', with: "andrewfink@gmail.com")
+    fill_in('password', with: "password")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.